<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoFileConfigWindowClass</class>
 <widget class="QWidget" name="VideoFileConfigWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoFileConfigWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Frames Per Second:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fpsLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Total Frames:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalFramesLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Current Frame:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentFrameLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>FrameSlider:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="frameSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
